src/App.tsx

```diff
@@ -1,39 +1,61 @@
 import React from 'react';
 import './App.scss';
-import Carousel from './components/Carousel';
+import Carousel from './components/Carousel/Carousel';
+import { useState } from 'react';
+// eslint-disable-next-line max-len
+import { OptionForCarousel } from './components/OptionForCarousel/OptionForCarousel';
 
-interface State {
+export interface State {
   images: string[];
 }
 
-class App extends React.Component<{}, State> {
-  state = {
+const App: React.FC<{}> = () => {
+  const [frameSize, setFrameSize] = useState(3);
+  const [step, setStep] = useState(1);
+  const [prev, setPrev] = useState(1);
+  const [itemWidth, setItemWidth] = useState(130);
+  const [animationDuration, setAnimationDuration] = useState(1000);
+  const [visibleItem] = useState<State>({
     images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
+      `./img/1.png`,
```

Comment:
```suggestion
      './img/1.png',
```
it can be the regular string